src/main/java/chatty/app/ChattyBot.java

```diff
@@ -0,0 +1,144 @@
+package chatty.app;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import chatty.exceptions.ChattyException;
+import chatty.exceptions.EmptyDescriptionException;
+import chatty.exceptions.MalformedArgumentsException;
+import chatty.parser.Parser;
+import chatty.storage.Storage;
+import chatty.task.Deadline;
+import chatty.task.Event;
+import chatty.task.Task;
+import chatty.task.TaskList;
+import chatty.task.Todo;
+import chatty.ui.Ui;
+
+/**
+ * Entry point for the ChattyBot application.
+ * This class is responsible for initializing the application and handling the main event loop.
+ * It uses the Ui, Storage, and TaskList classes to manage user interactions, data persistence, and task management.
+ *
+ * @author K0p1-Git
+ */
+public class ChattyBot {
+
+    private final Ui ui;
+    private final Storage storage;
+    private final TaskList tasks;
+
+    /**
+     * Constructs a new ChattyBot instance.
+     * Initializes the user interface, storage, and task list.
+     * Loads any existing tasks from storage.
+     */
+    public ChattyBot() {
+        this.ui = new Ui();
+        this.storage = new Storage();
+        ArrayList<Task> seed = storage.load();
+        this.tasks = new TaskList(seed);
+    }
+
+    /** Runs the main event loop of ChattyBot. */
```

Comment:
Should the opening /** be on a separate line? 
```suggestion
    /** 
     * Runs the main event loop of ChattyBot. 
     */
```